app/src/main/res/layout/layout_reviewable_list.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".activity.classrooms.ClassroomsListActivity"
+    android:orientation="vertical">
+
+    <include layout="@layout/reviewable_controls"
+        android:id="@+id/reviewableListControls"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="16dp"/>
+
+    <androidx.recyclerview.widget.RecyclerView
```

Comment:
A little suggestion: While you're at it, I think we could improve the user experience by displaying a little scrollbar to see where we are in the list while scrolling. This can be done by adding `android:scrollbars="vertical"` (also applies for the list of reviews of course).